images/runtime/python/3.10/bullseye.Dockerfile

```diff
@@ -66,6 +66,12 @@ ENV PATH="/opt/python/3/bin:${PATH}"
 # Bake Application Insights key from pipeline variable into final image
 ARG AI_KEY
 ENV ORYX_AI_INSTRUMENTATION_KEY=${AI_KEY}
+#Bake in client certificate path into image to avoid downloading it
```

Comment:
These python changes are not related to this update, I assume they are from running the build runtime images commands that runs the generate dockerfiles scripts

Reply:
Yes, seems likely indeed. Tagging @snehapar9 and @cormacpayne just in case they see any issues with it.